api/client.go

```diff
@@ -71,7 +74,12 @@ func (c *Client) Do(req *http.Request, v interface{}) (*http.Response, error) {
 	default:
 		if v != nil {
 			defer res.Body.Close()
-			if err := json.NewDecoder(res.Body).Decode(v); err != nil {
+
+			var bodyCopy bytes.Buffer
+			body := io.TeeReader(res.Body, &bodyCopy)
+
+			if err := json.NewDecoder(body).Decode(v); err != nil {
+				debug.Println("res.Body:", bodyCopy.String())
 				return nil, fmt.Errorf("error parsing API response - %s", err)
```

Comment:
We should change this message to not return the error from the JSON decoder and add that error to the debug output. This message could be much friendlier and direct the user to either re-run the command with debug output or to file a ticket. If we get a request-id on the response back, we should display that as part of the friendly message. Thoughts?